app/assets/stylesheets/common/base/_topic-list.scss

```diff
@@ -127,7 +127,7 @@
 
 // Topic list body
 
-.topic-list-item.visited,
+.topic-list-item.visited .topic-list-data,
```

Comment:
Nitpicking (but could be important because CSS specificity matters), you like don't need this change (the other selectors here also target the table rows).

Reply:
I found through testing I needed to add the specificity here, because once a topic was visited, these changed were not being applied to visited rows. Adding `topic-list-data` to be more specific solved the issue.

Reply:
Hmm, that shouldn't be the case. Do you mean there is a separate regression here?

Reply:
Oh I think you are right. `visited` is a class added to `topic-list-item`. Here we are checking for `visited` to be on the link, which I dont believe it is anymore.

Reply:
For instance, Im not sure these lines apply to anything https://github.com/jordanvidrine/discourse/blob/0e371d4c6dbd0f3d30e6ac5c17de5ead726ffa5e/app/assets/stylesheets/common/base/_topic-list.scss#L163-L171